lib/cfg-grammar.y

```diff
@@ -1079,6 +1085,33 @@ string_or_number
         | LL_FLOAT                              { $$ = strdup(lexer->token_text->str); }
         ;
 
+existing_path
+	: string
+	  {
+            struct stat buffer;
+            int ret = stat($1, &buffer);
+            CHECK_ERROR((ret == 0), @1, "File \"%s\" does not exist: %s", $1, strerror(errno));
+	    g_string_append_printf(configuration->file_list, "%s\n", $1);
```

Comment:
The previous version was better. file_list should be a list. Now the visualization mechanics and the internal representation is intertwined. For example, newline is a valid file name character in linux, which is difficult to handle with this change.

Reply:
ok. I will revert back to the previous version.